lab-davide/gulpfile.js

```diff
@@ -0,0 +1,61 @@
+var gulp = require('gulp');
+var mocha = require('gulp-mocha');
+var gulp =  require('gulp')
+
+gulp.task('default', function() {
+  return gulp.src('./test/greet-test.js', {read: false})
```

Comment:
Instead of specifying the exact test file to run, I suggest using a wildcard operator (*) to make it so all the test files in your test folder run automatically.  This way if you add more tests they are automatically run by your gulp testing without you needing to add them to your gulp.src here explicitly.  It would be like this:

'./test/*.js' instead of './test/greet-test.js'